src/licensedcode/plugin_license.py

```diff
@@ -139,6 +140,15 @@ class LicenseScanner(ScanPlugin):
             help_group=SCAN_OPTIONS_GROUP,
         ),
 
+        PluggableCommandLineOption(
+            ('--external-license-directory',),
+            required_options=['license'],
+            multiple=True,
+            type=click.Path(exists=True, readable=True, path_type=str),
+            help='Include additional directories for license detection.',
```

Comment:
You use additional here again. So IMHO additional is the right term to use throughout, not external.
Also I would add a `metavar='DIR'` and the help could be refined.... What about:
`Include this directory with additional custom licenses and license rules in the license detection index.`